src/Tasks/CombineTargetFrameworkInfoProperties.cs

```diff
@@ -36,14 +32,15 @@ public override bool Execute()
         {
             if (PropertiesAndValues != null)
             {
-                XElement root = new XElement(RootElementName);
-
+                StringBuilder sb = StringBuilderCache.Acquire();
+                sb.AppendLine($"<{RootElementName}>");
                 foreach (var item in PropertiesAndValues)
                 {
-                    root.Add(new XElement(item.ItemSpec, item.GetMetadata("Value")));
+                    sb.AppendLine($"  <{item.ItemSpec}>{item.GetMetadata("value")}</{item.ItemSpec}>");
```

Comment:
Is the getmetadata search case sensitive? And should there be a tab in the append line?

Reply:
It should be case-insensitive, but there's no reason to test that here...

I had two spaces. That seemed to be how it was with XElement rather than the tab when I was inspecting its escaped form.

Reply:
Also, I verified that this can build the repro but not that it actually produces correct results, which is why I said it isn't properly tested yet 🙂 In my list of things to do.